server/data/incentivesApi.ts

```diff
@@ -153,6 +153,13 @@ export class IncentivesApi extends RestClient {
     })
   }
 
+  setIncentiveLevelOrder(levelCodes: string[]): Promise<IncentiveLevel[]> {
+    return this.patch({
+      path: `/incentive/level-order`,
+      data: levelCodes as unknown as Record<string, unknown>,
```

Comment:
Wasn't this endpoint getting a list of Incentive levels codes, e.g. a `string[]`?

What's with the `Record<string, unknown>`?

Reply:
yeah the api endpoint wants `string[]` but the superclass of this thing does not explicitly allow `any` or `unknown`. what it does do is provide a default value of `{}` which TS decides is `Record<string, unknown>`.

what i don't quite get is why `Omit<Partial<IncentiveLevel>, 'code'>` satisfies `Record<string, unknown>` but `IncentiveLevel` does NOT satisfy `Record<string, unknown>`! they both have properties which are named only by strings and they both have values of some sort